src/Tasks.php

```diff
@@ -428,12 +428,13 @@ function run($opts = ['port' => 8088]) {
    *
    * @option boolean install Trigger an install on Pantheon.
    * @option boolean y Answer prompts with y.
+   * @option string pantheon-branch Use specified branch.
```

Comment:
Yeah, and this code too -- let's make it clear what the branch means, because even thinking it through I'm not sure. We're deploying to this branch on Pantheon? Or we're deploying from this branch on Github?